client/app/templates/json-api-errors.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if errors}}
```

Comment:
Ha, cool nice. That will do just fine, I think we can do that then upgrade to component if there is any processing to be done on the errors...

Cool, my next commit is going to be registration stuff which will use it so I will clean that up in there.